platforms/android/tangram/build.gradle

```diff
@@ -45,6 +45,8 @@ android {
     }
   }
 
+  flavorDimensions "tangramAbi"
```

Comment:
With the "variant-aware dependency management" in the new plugin, I'm no longer sure that product flavors are the right way for us to manage ABI variants. 

From the perspective of an application that depends on `tangram`, all variants of the library should provide all available ABIs. This is because ABI variants in applications are typically not managed with product flavors, but with [separate APKs](https://developer.android.com/studio/build/configure-apk-splits.html). Using flavors for this wasn't previously a problem because flavors weren't exposed when the library was used as a dependency.

Moving forward, I think we should put all of our supported ABIs in the `defaultConfig` and not separate them by flavor. There's clearly a benefit to building only a single ABI during a development for a fast build/run cycle - and luckily for us a feature has snuck into Android Studio that does just that. When a run configuration is started from Studio, it will build only the ABIs needed for the deployment device (see [this thread](https://code.google.com/p/android/issues/detail?id=221098#c20) for discussion).